docs/json_rpc_api.md

```diff
@@ -566,6 +567,18 @@ Example Return|`{`<br />&nbsp;&nbsp;`"bytes": 310768,`<br />&nbsp;&nbsp;`"size":
 |Example Return|`true`|
 [Return to Overview](#MethodOverview)<br />
 
+***
```

Comment:
Along the same lines, this doco should move to section 6 where the extension methods are defined.